certbot-dns-rfc2136/certbot_dns_rfc2136/__init__.py

```diff
@@ -0,0 +1,92 @@
+"""
+The `~certbot_dns_rfc2136.dns_rfc2136` plugin automates the process of
+completing a ``dns-01`` challenge (`~acme.challenges.DNS01`) by creating, and
+subsequently removing, TXT records using RFC 2136 Dynamic Updates.
+
+
+Named Arguments
+---------------
+
+===================================== =====================================
+``--dns-rfc2136-credentials``         RFC 2136 credentials_ INI file.
+                                      (Required)
+``--dns-rfc2136-propagation-seconds`` The number of seconds to wait for DNS
+                                      to propagate before asking the ACME
+                                      server to verify the DNS record.
+                                      (Default: 60)
+===================================== =====================================
+
+
+Credentials
```

Comment:
I was thinking about doing that until I had 4% battery left. I've added that now.